src/Template/Bake/Template/index.twig

```diff
@@ -42,7 +42,7 @@
         <thead>
             <tr>
 {% for field in fields %}
-                <th scope="col"><?= $this->Paginator->sort('{{ field }}') ?></th>
+                <th scope="col"><?= $this->Paginator->sort('{{ field }}', ['label' => __('{{ fieldHumanNames[field] }}')]) ?></th>
```

Comment:
This method doesn't have a label option in the second argument. See [the API docs](https://api.cakephp.org/3.5/class-Cake.View.Helper.PaginatorHelper.html#_sort).

Reply:
Hmm. But it seems to be working:
![kep](https://user-images.githubusercontent.com/1609182/33306373-9e06b7d8-d412-11e7-8869-3b1c37b21829.png)

What do you recommend? Is this an undocumented feature, which might change in the future?